app/src/main/res/layout-v26/autofill_cta_presentation.xml

```diff
@@ -12,6 +12,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:theme="@style/AppTheme"
+        android:background="@color/background_white"
```

Comment:
I don't think this is the right place to define the background since it creates the overdraw error. I would prefer it to be set in the `styles` or explicitly set when the autofill dialog is created.